tests/test_time.py

```diff
@@ -17,6 +17,9 @@ def test_time_spec():
 
     dt = datetime(2017, 11, 26, 15, 33)
     assert matches_time_spec(dt, 'Sat-Sun 15:30-16:00 UTC')
+    assert matches_time_spec(dt, 'Sat-Sun 15:30-08:00 UTC')
```

Comment:
isn't this a bit ambiguous? is the time range including 08:00 UTC on Monday or not?

Reply:
Yeah, as written it does include the hours after midnight on Monday morning. I agree it's ambiguous. What might be a better behaviour do you think?

Reply:
e.g. 'Wed-Sun 15:00-08:00 UTC' definitely could mean "between 3pm and 8am every day until 8am Sun". I'd be happy with that. If you are, I'll update the PR and include explanation in the README.

Reply:
My particular use-case is along the lines of "downscale (dev, test) from 6pm 'til 7am every weeknight", and it confused me why it never happened! :-)